locale/translations/fr_FR.json

```diff
@@ -254,6 +254,7 @@
     "form.prefs.select.older_first": "Ancien éléments en premier",
     "form.prefs.select.recent_first": "Éléments récents en premier",
     "form.prefs.label.keyboard_shortcuts": "Activer les raccourcis clavier",
+    "form.prefs.label.custom_css": "Custom CSS",
```

Comment:
Ideally, translations should be translated :) You can use use Google Translate or similar.